gateway/services/profile.go

```diff
@@ -61,6 +61,18 @@ var ProfileRoutes = arbor.RouteCollection{
 		"/profile/filtered/",
 		alice.New(middleware.AuthMiddleware([]models.Role{models.AdminRole, models.StaffRole}), middleware.IdentificationMiddleware).ThenFunc(GetFilteredProfiles).ServeHTTP,
 	},
+	arbor.Route{
+		"GetEventRedemptionStatus",
+		"GET",
+		"/profile/checkin/{id}/",
+		alice.New(middleware.AuthMiddleware([]models.Role{models.AdminRole}), middleware.IdentificationMiddleware).ThenFunc(GetEventRedemptionStatus).ServeHTTP,
+	},
+	arbor.Route{
+		"AwardPoints",
+		"GET",
```

Comment:
So I discussed this with someone else, and even though we're effectively sending data to the server for this processing, I was told it's not that bad to just leave it, especially since we're retrieving some new information about the status of the checkin.

Reply:
Changing this to a post.